packages/@ourworldindata/utils/src/owidTypes.ts

```diff
@@ -423,6 +423,7 @@ export interface OwidArticleContent {
     byline?: string | string[]
     dateline?: string
     excerpt?: string
+    toc?: TocHeading[]
```

Comment:
Once we have the typed components branch merged in I think we should consider not storing the (duplicated) information of toc headings here on the document level that would be persistet. I think it would then be nicer to derive this from the body when needed. But for now I think this is totally fine

Reply:
yep that makes sense